spec/policies/person_policy_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe PersonPolicy do
+  subject { described_class.new(user, person) }
+
+  let(:person) { create :person }
+
+  context 'with admin access' do
+    let(:user) { create :user, :admin }
+
+    it { is_expected.to permit_actions(%i[index show create update destroy]) }
+  end
+
+  context 'with redactor access' do
+    let(:user) { create :user, :redactor }
+
+    it { is_expected.to permit_actions(%i[index show create update destroy]) }
+  end
+
+  context 'with reviewer access' do
+    let(:user) { create :user, :reviewer }
+
+    it { is_expected.to permit_actions(%i[index show]) }
+    it { is_expected.to forbid_actions(%i[create update destroy]) }
```

Comment:
does rubocop don't say anything about 2 it in one context?

Reply:
![image](https://user-images.githubusercontent.com/70864999/156394079-b6f966e3-0d7d-4769-88f1-c0452285321b.png)